src/bsp/platform/esp32/include/xi_esp32_sft_notifications.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2003-2017, LogMeIn, Inc. All rights reserved.
+ *
```

Comment:
I'd rename this to `xi_bsp_fwu_notifications_esp32.h` or `xi_bsp_fwu_notifications.h`.